instrumentation/manticore/lib/opentelemetry/instrumentation/manticore/instrumentation.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Manticore
+      # The Instrumentation class contains logic to detect dependencies and install the Manticore instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
```

Comment:
Per my Other comment, Let’s add a ‘compatible’ block here for checking platform, wdyt?

Reply:
line 13 has something like that. Or do you have a better example? 

 The builds that are failing are due to gem not found when trying to install, it doesn't reach into this code yet.

Reply:
I see, fair enough, I’ll have to review. I think we’re definitely lacking in a comprehensive way to determine platform compatibility, but ideally can provide something short term here to workaround while we make this a bit more robust long term

Reply:
Let me know how I help if needed. Thanks!